modules/ossm-vs-istio.adoc

```diff
@@ -126,7 +126,16 @@ You can deploy virtual machines to OpenShift using OpenShift Virtualization. The
 [id="ossm-gateways_{context}"]
 == Gateways
 
-{SMProductName} installs ingress and egress gateways by default. You can disable this in the SMCP using `spec.gateways.ingress.enabled=false` or `spec.gateways.egress.enabled=false`.
+{SMProductName} installs ingress and egress gateways by default. You can disable gateway installation in the `ServiceMeshControlPlane` (SMCP) resource using the following settings:
+
+* `spec.gateways.enabled=false` to disable both ingress and egress gateways.
+* `spec.gateways.ingress.enabled=false` to disable ingress gateways.
+* `spec.gateways.egress.enabled=false`  to disable egress gateways.
+
+[NOTE]
+====
+The default gateways are annotated to indicate that they are generated by and managed by the {SMProductName} Operator.
```

Comment:
A suggestion is to change 'are annotated' and 'are generated to the active voice, if possible to retain the meaning.

See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#voice